tests/unit/test_renamed_relations.py

```diff
@@ -0,0 +1,17 @@
+from dbt.contracts.relation import RelationType
```

Comment:
This line was manually updated from `from dbt.adapters.contracts.relation import RelationType`. This class was moved during the decoupling of `dbt-core` and `dbt-adapters`, which happened in `1.8`.